code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -473,6 +473,13 @@
 	required_reagents = list(/datum/reagent/consumable/laughter = 1, /datum/reagent/consumable/cream = 1)
 	reaction_tags = REACTION_TAG_DRINK | REACTION_TAG_EASY | REACTION_TAG_OTHER
 
+/datum/chemical_reaction/drink/bureau
```

Comment:
I would like this in the monkestation module folder instead.
'monkestation/code/modules/food_and_drinks/recipes/drinks_recipes.dm
Also try to put new things at the bottom instead of in the middle.